### ISSUE TITLE
Importance to update the lambda slowly

### ISSUE THREAD

**Issue by wang3702** [YU1ut/MixMatch-pytorch#9]
Hi, I have tested on  your original code and my reproduced code. I found the accuracy will increase more smoothly as Google suggested to update lambda to 72 with 1024 epochs. Here is the comparison of our result in first around 100 epochs:

![mixmatch_yu](https://user-images.githubusercontent.com/50850224/59525919-7d48cf00-8e8c-11e9-8069-1643d15f3410.PNG)
My training result:
![Accuracy](https://user-images.githubusercontent.com/50850224/59525938-88036400-8e8c-11e9-959b-14eeefa5ffe9.jpg)
Loss comparison:
![mixmatch_yu1](https://user-images.githubusercontent.com/50850224/59526121-08c26000-8e8d-11e9-9310-4ce814eb8e08.PNG)
My training result
![CLoss](https://user-images.githubusercontent.com/50850224/59525947-8cc81800-8e8c-11e9-82d1-54d1bf35d974.jpg)

**Comment by JongMokKim** [comment]
hello! what does that mean  'update lambda to 72 with 1024 epochs' ? 
you mean, max lambda value is 72 and gradually increase in whole epoch (1024)? 
thank you in advance!

**Comment by wang3702** [comment]
@JongMokKim Yes. Exactly. Slowly update is much better for stable training

**Comment by SanyouWu** [comment]
> @JongMokKim Yes. Exactly. Slowly update is much better for stable training

So, how to design the function to control the change process of lambda?

**Comment by YU1ut** [comment]
Updated the calculation of lambda to make it gradually increase in the whole epoch.
